i18n_translation/i18n_translation.module

```diff
@@ -136,9 +136,9 @@ function i18n_translation_mode_element($object_type, $i18n_mode = I18N_MODE_NONE
 /**
  * Get list of translation modes.
  *
- * @param $container_type
+ * @param object $container_type
```

Comment:
That's not an object. It can be a string and possibly an array (I'm not 100% sure).